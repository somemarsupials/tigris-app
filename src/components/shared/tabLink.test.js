import React from 'react';
import { shallow } from 'enzyme';
import TabLink from './tabLink';

describe('TabLink', () => {
  it('matches snapshot', () => {
    expect(shallow(<TabLink />)).toMatchSnapshot();
  });
});

