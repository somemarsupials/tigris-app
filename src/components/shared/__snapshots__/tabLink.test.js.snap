// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabLink matches snapshot 1`] = `
<a
  onClick={[Function]}
/>
`;
